### PR TITLE
CGMTA ingestion

### PR DESCRIPTION
**Pull request by anthology-assist**

### CODE REVIEW

**Comment by alumae** [comment]
As an editor for CGMTA, I conform that the preview site https://preview.aclanthology.org/CGMTA-ingestion-2023/2023.cgmta-1.0/ looks OK.

File: data/xml/2023.cgmta.xml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      <booktitle>Proceedings of the NoDaLiDa 2023 Workshop on Constraint Grammar - Methods, Tools and Applications</booktitle>
+      <editor><first>Eckhard</first><last>Bick</last></editor>
+      <editor><first>Trond</first><last>Trosterud</last></editor>
+      <editor><first>Tanel</first><last>Alumäe</last></editor>
+      <publisher>Association of Computational Linguistics</publisher>
+      <address>Tórshavn, Faroe Islands</address>
+      <month>May</month>
+      <year>2023</year>
+      <url hash="97ad1022">2023.cgmta-1</url>
+      <venue>cgmta</venue>
```
Comment (mjpost) [inline data/xml/2023.cgmta.xml:14]: Add ws venue

**Review by mjpost — COMMENTED** [review]
Please link to #2857 in the "Development" sidebar

File: data/xml/2023.cgmta.xml (hunk 1 above)
Comment (anthology-assist) [reply data/xml/2023.cgmta.xml:14]: Done.

**Comment by mjpost** [comment]
@alumae Is this an workshop that is likely to happen again? Or is it just a one-time workshop hosted by nodalida?

**Review by mjpost — APPROVED** [review]

**Pull request merged**